crates/rpc/src/routes/beacon.rs

```diff
@@ -28,7 +28,20 @@ pub fn get_beacon_routes(
         .and_then(move || get_genesis(network_spec.genesis.clone()))
         .with(log("genesis"));
 
+    let fork = {
+        let db = db.clone();
+        beacon_base
+            .and(path("states"))
+            .and(param::<ID>())
+            .and(path("fork"))
+            .and(end())
+            .and(get())
+            .and_then(move |state_id: ID| get_fork(state_id, db.clone()))
```

Comment:
```suggestion
            .and_then(move |state_id: ID| get_fork(state_id, db))
```

we don't need the clone here